src/blacksmith/sd/_sync/adapters/nomad.py

```diff
@@ -0,0 +1,27 @@
+"""
```

Comment:
Do not look for a file to refer in the doc for the develop eversion of the sync.

It is not refered since those file are generate by unasync.